api/spec/v1.yaml

```diff
@@ -2058,6 +2058,49 @@ components:
         Values of EVM type `address` and `address payable` are represented as lowercase hex strings with a "0x" prefix.
         Values of EVM type `bytes` and `bytes<N>` are represented as base64 strings.
         Values of other EVM types (integer types, strings, arrays, etc.) are represented as their JSON counterpart.
+      
+    RuntimeEvmContract:
+      type: object
+      properties:
+        creation_tx:
+          type: string
+          description: | 
+            The Oasis cryptographic hash of the transaction that created the smart contract.
+             Can be omitted for contracts that were created by another contract, as opposed 
+             to a direct `Create` call.
+        creation_bytecode:
+          type: string
+          format: byte
+          description: |
+            The creation bytecode of the smart contract. This includes the constructor logic
+            and the constructor parameters. When run, this code generates the runtime bytecode.
```

Comment:
Similarly as `creation_tx` this can be omitted right if deployed from another contract?